### ISSUE TITLE
Add an input option to select a specific proxy provider

### ISSUE THREAD

**Issue by jancurn** [apifytech/actor-google-search-scraper#27]
The reality is, each underlying proxy provider sometimes returns slightly different results. We should provide users with the ability to select the provider.

**Comment by jancurn** [comment]
Actually, a cleaner and more generic way to do this is to add an input field called **Additional query parameters** that will be appended to the URL. We can add a special param to pick the provider, or use this for other customizations.

**Comment by metalwarrior665** [comment]
This would be great for the focused paid ads scraping.

**Comment by jancurn** [comment]
BTW you can already do this now, using the URLs instead of search tersm and using the `xprovider` query parameter

**Comment by metalwarrior665** [comment]
Yes but it is not documented and the differences between providers are not explained.

Also, we had a use-case for screenshots where some providers show a pop-up.
